debugger/src/components/Debugger/Signals/Signal/index.js

```diff
@@ -110,7 +98,7 @@ export default connect({
     renderAction (action, index) {
       if (action._functionTreePrimitive) {
         return (
-          <div key={index}>
+          <div key={index} onClick={(event) => event.stopPropagation()}>
             <span className='signal-groupName'><strong>{action.type}</strong>{action.name ? ': ' + action.name : null}</span>
```

Comment:
@christianalfoni I think this is related:
https://facebook.github.io/react/docs/conditional-rendering.html#inline-if-with-logical--operator